src/go/rpk/pkg/cli/cmd/redpanda/start_test.go

```diff
@@ -1379,6 +1379,24 @@ func TestStartCommand(t *testing.T) {
 			"--install-dir", "/var/lib/redpanda",
 			"--sandbox",
 		},
+		postCheck: func(
+			fs afero.Fs,
+			rpArgs *redpanda.RedpandaArgs,
+			st *testing.T,
+		) {
+			require.Equal(st, "true", rpArgs.SeastarFlags["overprovisioned"])
+			require.Equal(st, "0M", rpArgs.SeastarFlags["reserve-memory"])
+			conf, err := new(config.Params).Load(fs)
```

Comment:
I think we can avoid checking errors like this for tests, fwiw -- this is checked in so many places that if we have a mistake setting up a conf for a test, we can panic.

no need to remove, just potentially for the future